web/pgadmin/tools/sqleditor/static/js/components/sections/Query.jsx

```diff
@@ -200,7 +198,7 @@ export default function Query({onTextSelect}) {
     eventBus.registerListener(QUERY_TOOL_EVENTS.SAVE_FILE, (fileName)=>{
       queryToolCtx.api.post(url_for('sqleditor.save_file'), {
         'file_name': decodeURI(fileName),
-        'file_content': editor.current.getValue(),
+        'file_content': editor.current.getValue(false, editor.current?.getEOL()),
```

Comment:
why need to pass eol? Editor has it in the memory right? You could pass a boolean flag - `useLineSep` and based on that you can decide in getValue to use EOL from state or not.

Reply:
Sure we can do that also. will update.